joomla/joomla-cms
# [4.1] Scheduled tasks notes

If a task has a note then it is displayed below the task name inside a small span (just like in all other components)

If a task does not have a note then a small span is still present but it contains a language string (which is empty)

There is no need for this empty span and there is certainly no need for a language string with no content.

## To test
Create a new task with a note
Create another new task without a note
Check the source code and you will see the empty span on the second task

Apply this PR
Refresh and check the source of the page again. This time there is no empty span on the task without a note and the note is still displayed on the other task.
